aries-site/src/examples/templates/table-customization/components/TableCustomizationExample.js

```diff
@@ -0,0 +1,250 @@
+import React, { useContext, useEffect, useRef, useState } from 'react';
```

Comment:
In mobile view, I would expect the table to have horizontal scroll, but the rest of the page is static. Right now, the whole page gets horizontal scroll:
<img width="477" alt="Screen Shot 2021-05-27 at 11 28 22 AM" src="https://user-images.githubusercontent.com/12522275/119878224-bbefbc80-bede-11eb-95ed-a98aa071dd5c.png">

Reply:
Makes sense. Fixed. I also pin the name column.